code/modules/robotics/bot/guardbot.dm

```diff
@@ -336,6 +365,9 @@
 
 	New()
 		..()
+#ifdef ASS_JAM
```

Comment:
```suggestion
#if ASS_JAM
```
As it is now this will always trigger. 😱

Reply:
Also this could be moved to the definition of lawbringer_alwaysbigshot instead of doing it in New.

Reply:
...that would explain a few things. Fixed it and moved it up to the var defs!